opencog/benchmark/AtomSpaceBenchmark.cc

```diff
@@ -68,14 +70,16 @@ AtomSpaceBenchmark::AtomSpaceBenchmark()
     doStats = false;
     testKind = BENCH_AS;
 
+    randomseed = (unsigned long) time(NULL);
```

Comment:
That was the only way it was done before. See lines 77 and 78. I added the ability to use a fixed randseed. It didn't exist before. I copied this setting based on time() from the 

I do find it useful to see different random values at times so you can determine the algorithmic sensitivity to different sequences. Most tests aren't sensitive but the getHandlesOfType was.